docs/hub/model-release-checklist.md

```diff
@@ -88,6 +91,8 @@ Bonus: a recognised library also allows you to track downloads of your model ove
    
    Try this model directly in your browser: [Space Demo](https://huggingface.co/spaces/username/model-demo)
    ```
+   
+When you write a demo, please download the model from its repository in the Hub (instead of using external sources) so both are cross-linked together. This will help all your artifacts achieve greater visibility.
```

Comment:
```suggestion
When you create a demo, please download the model from its repository on the Hub (instead of using external sources like Google Drive); it cross-links model artefacts and demo together and allows more paths to visibility. 
```

unifying it with the line above.

Reply:
yeah, looks better, thanks!